doc/pbh/pbh-design.md

```diff
@@ -203,8 +212,10 @@ A custom hashing can be configured for Regular/FG ECMP and LAG.
 ###### Figure 2: PBH OA design
 
 A `PbhOrch` class with a set of data structures will be implemented to handle PBH feature.  
-OA will be extended with a new PBH Config DB schema and SAI FG Hash API support.  
+OA will be extended with a new PBH Config DB/State DB schema and SAI FG Hash API support.  
 PBH table/rule/hash/hash-field updates will be processed by OA based on Config DB changes.  
+Each update operation will be verified against vendor specific capabilities.  
```

Comment:
Is it SAI query API or pre-defined vendor capabilities or a mix of both? Could you please describe from where this capability data comes from?

Reply:
done